dist/profile/templates/kubernetes/resources/nginx/default-deployment.yaml.erb

```diff
@@ -14,17 +14,17 @@ spec:
       - name: default-http-backend
         # Any image is permissable as long as:
         # 1. It serves a 404 page at /
-        # 2. It serves 200 on a /healthz endpoint
-        image: gcr.io/google_containers/defaultbackend:1.0
+        # 2. It serves 200 on a / endpoint
+        image: olblak/jenkins404:3519ac
```

Comment:
This should be something we host under jenkinsciinfra, can you fork a repo into `jenkins-infra` so we're not relying on a personal container.

Reply:
Sure

Reply:
but I don’t have the permission to create repositories on jenkins-infra